module/hooks/talent.mjs

```diff
@@ -108,9 +225,37 @@ HOOKS.conserveeffort00 = {
 
 /* -------------------------------------------- */
 
-HOOKS.irrepressiblespi = {
-  startTurn(item, {resourceRecovery}) {
-    if ( !this.system.isBroken ) resourceRecovery.morale = (resourceRecovery.morale || 0) + 1;
+HOOKS.distancerunner00 = {
+  prepareMovement(_item, movement) {
+    movement.stride += 1;
+  }
+}
+
+/* -------------------------------------------- */
+
+HOOKS.dustbinder000000 = {
+  applyCriticalEffects(_item, action, outcome, _self) {
+    // Require Earth rune
+    if ( action.rune?.id !== "earth" ) return;
+
+    // Require damage to Health
+    const damageHealth = outcome.resources.health < 0;
+    if ( !damageHealth ) return;
+
+    // Apply Corroding effect
+    outcome.effects.push(SYSTEM.EFFECTS.corroding(this));
```

Comment:
I think - in hindsight - these hooks are probably unnecessarily commented/documented. The code is self-documenting and short enough. Not required, but you could trim some of these comment and whitespace lines if you like, for example:
```suggestion
    if ( action.rune?.id !== "earth" ) return;
    const damageHealth = outcome.resources.health < 0;
    if ( damageHealth ) outcome.effects.push(SYSTEM.EFFECTS.corroding(this));
```